backend/de.metas.business/src/main/java/de/metas/invoice/sequence/InvoiceCountryIdProvider.java

```diff
@@ -37,7 +37,7 @@
 
 public class InvoiceCountryIdProvider implements ICountryIdProvider
 {
-	private static final List<String> INVOICE_DOCBASETYPES = Arrays.asList("API", "APC", "ARI", "ARC");
+	private static final List<String> INVOICE_DOCBASETYPES = Arrays.asList("API", "APC", "ARI", "ARC", "CUI");
```

Comment:
instead of hardcoded string pls use org.compiere.model.X_C_DocType#DOCBASETYPE_*

Reply:
done